src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -140,6 +161,44 @@ PrefixModeStorage::PrefixModeStorage() {
   }
 }
 
+std::string PrefixModeStorage::prefix_from_proto_package_mappings(const FileDescriptor* file) {
+  if (!file) {
```

Comment:
We actually do need the `file` here as we perform some logic below to figure out the package to use.

Reply:
I think I had commented not about the passing of `file`, but the passing of it by reference so it can't be `nullptr`.

Reply:
ah